### DISCUSSION TITLE
Playground file structure

### DISCUSSION THREAD

**Discussion by darrylb123** [tasmota/Berry_playground#6] [Playground Management]
As a newbie Berry coder not so long ago, I found difficulty finding small examples of the Berry language for even simple constructs. The documentation is actually very good but examples are usually buried in complex applications. Even simple things like use a file autoexec.be is to auto-start is only found when looking through discussions.

I propose to create a directory of examples that follow the order of the Tasmota Berry documentation, 
Eg:
- Timers
- Custom Commands
- Custom Responses
- Relays and Lights
- etc

The example file should make use of each function in the group, eg show using all the timer functions, and work.

I am certainly not an accomplished coder but I would be happy to create something to start with. I have time (I'm retired) to work my way through the simpler API functions.

**Comment by dkebler** [comment]
see #10
